prh/ja.yml

```
@@ -53,3 +53,6 @@ rules:
   - expected: 接尾辞
     patterns:
       - サフィックス
+  - expected: インターフェイス
+    patterns:
+      - インターフェース
```

JIS 表記も修正するようにお願いします。

```suggestion
      - インターフェース
      - インタフェース
```